### ISSUE TITLE
NetworkX graphs to TUD dataset format?

### ISSUE THREAD

**Issue by johnnytam100** [pyg-team/pytorch_geometric#4169]
### 🚀 The feature, motivation and pitch

Hi, just want to know if it already exists for a utility that converts NetworkX graphs to the TUD dataset format?
Thank you!

**Comment by rusty1s** [comment]
Do you mean the TUDataset output format, i.e. the text files? We don‘t have support for this yet, but it should be trivial to add (let me know if you are interested in contributing this). Nonetheless, I think we should allow this for PyG `data` objects rather than for networkx graphs.

**Comment by johnnytam100** [comment]
@rusty1s Yes, you're right, the TUDataset format -> I have a custom dataset consisting of multiple networkx graphs and would like to perform graph classification with PyG. Do you know a workaround that doesn't require the conversion of the networkx graphs to TUDataset so they can be fit into PyG? 

Besides, I will let you know if I come up with a well-written function of `networkx_to_tud`

**Comment by rusty1s** [comment]
You can simply convert the networkx graphs to PyG `data` objects by yourself using `torch_geometric.utils.from_networkx`. This way, you do not need to convert to the `TUDataset` format in the first place.
